apps/src/code-studio/pd/application/teacher1920/Section4ProfessionalLearningProgramRequirements.jsx

```diff
@@ -296,51 +212,25 @@ export default class Section4SummerWorkshop extends LabeledFormComponent {
         <div id="assignedWorkshops">
           {this.renderAssignedWorkshopList()}
         </div>
-
-        {this.isUnableToAttendAssignedWorkshop() &&
-        <div style={styles.indented}>
-          <p style={styles.formText}>
-            <strong>
-              We strongly encourage participants to attend their assigned summer workshop
-              (based on the region
-              in which you currently teach), so that you can meet the other teachers,
-              facilitators, and
-              Regional Partners with whom you will work in 2019-20.
-            </strong>
-          </p>
-          {this.renderAlternateWorkshopList()}
-        </div>
-        }
-        <p>
-          Teachers in this program are required to participate in both:
-        </p>
-        <ul>
-          <li>
-            One five-day, in-person summer workshop in 2019
-          </li>
-          <li>
-            Up to four one-day, in-person local workshops during the 2019-20 school year
-            (typically held on Saturdays)
-          </li>
-        </ul>
-        {this.radioButtonsWithAdditionalTextFieldsFor('committed', {
-          [TextFields.noExplain]: 'other'
-        })}
         <div>
           <label>
             Your application has been assigned to a program hosted by one of our Regional
             Partners based on your geographic location. There may be a fee associated with
             the program in your region. There also may be scholarships available to help
             cover the cost of the program. You can check{' '}
-            <a href="https://code.org/educate/regional-partner/summer-workshop-fee" target="_blank">
+            <a
+              href="https://code.org/educate/regional-partner/summer-workshop-fee"
+              target="_blank"
+            >
               this page to see if there are
             </a>
             {' '}fees and/or scholarships available in your region.
           </label>
           {this.radioButtonsFor("payFee")}
           {this.props.data.payFee === TextFields.noPayFee1920 && this.largeInputFor('scholarshipReasons')}
           {this.radioButtonsFor('willingToTravel')}
-          We may offer online academic year workshops for those unable to travel to their local academic year workshops. Important notes.
+          We may offer online academic year workshops for those unable to travel to their
+          local academic year workshops. Important notes.
```

Comment:
I know you didn't write this part, but shouldn't it be `Important notes:` with a colon it rather than a period?